cmd/core-command/main.go

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright 2018 Dell Inc.
```

Comment:
I propose [SPDX](https://spdx.org/) style headers, to keep consistency across the repo

Reply:
We addressed the format of the header before with LF.  Keeping it the same for now per agreement of community

Reply:
@jpwhitemn LF proposed SPDX (while I was proposing no comments at all). So we moved to SPDX with `export` services.

This is less important as a remark, I have no objection that comments are kept by a preference Just that I suggested code style aligning.